hcfb-be/server.js

```diff
@@ -35,44 +35,6 @@ var server = app.listen(8080, function () {
 })
  
 function initial(){
- 
-  // let customers = [
-  //   {
-  //     firstname: "Joe",
-  //     lastname: "Thomas",
-  //     age: 36
-  //   },
-  //   {
-  //     firstname: "Peter",
-  //     lastname: "Smith",
-  //     age: 18
-  //   },
-  //   {
-  //     firstname: "Lauren",
-  //     lastname: "Taylor",
-  //     age: 31
-  //   },
-  //   {
-  //     firstname: "Mary",
-  //     lastname: "Taylor",
-  //     age: 24
-  //   },
-  //   {
-  //     firstname: "David",
-  //     lastname: "Moore",
-  //     age: 25
-  //   },
-  //   {
-  //     firstname: "Holly",
-  //     lastname: "Davies",
-  //     age: 27
-  //   },
-  //   {
-  //     firstname: "Michael",
-  //     lastname: "Brown",
-  //     age: 45
-  //   }
-  // ]
```

Comment:
Is it worth keeping these around commented out?

Ideally, we could move them to test fixtures or development seed data or something...

Reply:
No we can delete those.

Reply:
Yeah we should be deleting this, if anyone is editing code and catches em go ahead and delete